model/src/main/kotlin/OrtResult.kt

```diff
@@ -62,8 +62,8 @@ data class OrtResult(
         val data: CustomData = emptyMap()
 ) {
     /**
-     * Return the concluded licenses associated to their package. If [omitExcluded] is set to true, excluded packages
-     * are omitted from the result.
+     * Return the concluded licenses for each package. If [omitExcluded] is set to true, excluded packages are omitted
```

Comment:
Commit message nits:
* woring -> wording
* not -> note

Reply:
Done.